Cargo.toml

```diff
@@ -71,7 +71,7 @@ fluvio-async-tls = { version = "0.3.0", optional = true }
 async-std = { version = "1.6.2", default-features = false, optional = true }
 async-io = { version = "1.1.2", optional = true }
 async-net = { version = "1.6.0", optional = true }
-memmap = { version = "0.7.0", optional = true }
+memmap = { package = "memmap2", version = "0.5", optional = true }
```

Comment:
I think that maybe we should update this to 
```suggestion
memmap2 = { version = "0.5", optional = true }
```

Reply:
so we should re-export it as `memmap2` as well?

Reply:
not sure since it is fork of the same crate

Reply:
I think that we should basically update all `memmap` references to `memmap2`. Unless we explicitly expose them in the public api

Reply:
So, I think that we should update it in https://github.com/infinyon/future-aio/blob/e97957ef12d294ef5dad06e8cfd0812a83bbeb1f/src/fs/mmap.rs#L13 and also in https://github.com/infinyon/future-aio/blob/e97957ef12d294ef5dad06e8cfd0812a83bbeb1f/Cargo.toml#L49

Reply:
yeah makes sense, done